examples/src/test/scala/org/virtuslab/ideprobe/ModuleTest.scala

```diff
@@ -0,0 +1,53 @@
+package org.virtuslab.ideprobe
+
+import java.nio.file.Files
+
+import org.junit.Assert
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ValueSource
+import org.virtuslab.ideprobe.protocol.{BuildScope, JUnitRunConfiguration, ModuleRef, Setting}
+import org.virtuslab.intellij.scala.SbtTestSuite
+import org.virtuslab.intellij.scala.protocol.SbtProjectSettingsChangeRequest
+
+class ModuleTest extends SbtTestSuite {
+  @ParameterizedTest
+  @ValueSource(
+    strings = Array("projects/shapeless/ideprobe.conf", "projects/cats/ideprobe.conf", "projects/dokka/ideprobe.conf")
```

Comment:
probably nicer to read if each array element is in new line

Reply:
done